docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -33,7 +33,24 @@ You will need storage for the backups. This topic shows the use of AWS S3 bucket
 
 * If you'll use AWS S3, before proceeding with the configuration described below, verify that you know the `aws_access_key_id` and `aws_secret_access_key` values. Or  contact your IT team if they manage those assets. You'll provide those details in an edited version of the [medusa-bucket-key.yaml](./medusa-bucket-key.yaml) file. For information about the S3 setup steps, see this helpful [readme](https://github.com/thelastpickle/cassandra-medusa/blob/master/docs/aws_s3_setup.md).  
 
-* If you haven’t already, install the k8ssandra chart.
+* If you haven't already, add and update the relevant repos:
+
+```
+helm repo add k8ssandra https://helm.k8ssandra.io/
```

Comment:
You only need this first line:

```
helm repo add k8ssandra https://helm.k8ssandra.io/
```

All of the charts are in that repo.

Reply:
Thanks - incorporating.